sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/ITokenStorage.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.appcenter.utils.context;
```

Comment:
Can you move the token storage classes to the Identity module? It doesn't need to exist in core

Reply:
We try to avoid coupling the modules as much as possible, and this puts more Identity logic in the core module than necessary

Reply:
It needs to exist in core because we share it with data sync.

Reply:
Doesn't data sync require identity module as a dependency? And in that case, why would data sync need to access storage, it can just use context

Reply:
Identity would retrieve token from storage and populate context

Reply:
Ah so you want to move only what can be moved and keep enough code in core just for sharing the token?

Reply:
Yep. I agree that context must be in core. But the token persistence/storage can be moved

Reply:
Sounds good.

Reply:
Discussed this with @achocron, all clear now